### PR TITLE
Text length classification

### PR DESCRIPTION
**Pull request by LeonardPuettmannKern**
PR checklist:
- [x] Tested by creator on localhost:8000/docs
- [x] Tested by creator on refinery
- [x] Tested by reviewer on localhost:8000/docs
- [x] Tested by reviewer on refinery
- [x] (If added) common code tested in notebook/ script
- [ ] Conforms with the agreed upon code pattern

### CODE REVIEW

File: classifiers/reference_quality/text_length_classifier/config.py [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+        available_for=["refinery", "common"],
+        part_of_group=[
+            "reference_quality",
+            "text_analysis"
+        ],  # first entry should be parent directory
+        # mapping lables for cognition
+        cognition_init_mapping={
+            "short": "Needs fix",
+            "medium": "null",
+            "long": "null",
```
Comment (JWittmeyer) [inline classifiers/reference_quality/text_length_classifier/config.py:24]: this shouldn't be a null string but the value None

**Comment by JWittmeyer** [comment]
wrong base branch?

**Comment by JWittmeyer** [comment]
branch names should be with -

**Comment by JWittmeyer** [comment]
A bit unsure about the split behavior. Curranty we would split on every whitespace and count the resulting array. Since the name is Text length classification i'd assume something different (since `this-is-a-long-text-but-i-dont-know-how-to-whitespace!` is short and `a, b, c &  d are chars` is medium)

Maybe wordcount to make it a bit more clear?

File: classifiers/reference_quality/text_length_classifier/config.py (hunk 1 above)
Comment (JWittmeyer) [reply classifiers/reference_quality/text_length_classifier/config.py:24]: not sure if it's possible though :D please test, if not ill think of something on the interpreter side

Comment (LeonardPuettmannKern) [reply classifiers/reference_quality/text_length_classifier/config.py:24]: Tested it and works fine. The attached picture shows how this is then stored in the CMS. This should be the same as doing a `json.dump(None)`, if I am not mistaken.

cms

Comment (JWittmeyer) [reply classifiers/reference_quality/text_length_classifier/config.py:24]: works for me

**Pull request closed without merging**